shared/constants/hardware-wallets.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * Accounts can be instantiated from simple, HD or the multiple hardware wallet
+ * keyring types. Both simple and HD are treated as default but we do special
+ * case accounts managed by a hardware wallet.
+ */
+export enum HardwareKeyringTypes {
+  ledger = 'Ledger Hardware',
+  trezor = 'Trezor Hardware',
+  lattice = 'Lattice Hardware',
+  qr = 'QR Hardware Wallet Device',
+  hdKeyTree = 'HD Key Tree',
+  imported = 'Simple Key Pair',
```

Comment:
Why were these moved here? This is now very misleading. These last two are not hardware wallets.